front_end/Matching/ResponseList.js

```diff
@@ -0,0 +1,189 @@
+        this.handleChange = this.handleChange.bind(this);
+        this.handleSend = this.handleSend.bind(this);
+    }
+
+    handleChange(value){
+        this.setState({reply: value});
+        if(value === ""){
+            this.setState({disabled: true});
+        } else{
+            this.setState({disabled: false});
+        }
+    }
+
+    handleSend(){
+        console.log("Press the send button!");
+        this.setState({sent: true, expand: false});
+    }
+
+    handleViewResponse(){
+        console.log("Viewing Response");
+    }
+
+    render(){
+        const send_color = this.state.disabled ? '#bfbfbf' : '#89cf50';
+        return(
+            <View style={styles.response__container}>
+                <View style={styles.response_header}>
+                    <Icon name='person-circle-sharp' size={30}/>
+                    <Text style={styles.username}>{this.state.username}</Text>
+                </View>
+                <View style={{marginBottom: 16}}>
+                    <ReadMore numberOfLines={4}>
+                        <Text style={styles.text}>{this.state.response}</Text>
+                    </ReadMore>
+                </View>
+                { this.state.sent ? 
+                    <TouchableOpacity style={styles.message_sent}>
+                        <Text style={styles.message_sent_text}>Sent! View Your Response&gt;</Text>
+                    </TouchableOpacity> 
+                    : <TextInput style={this.state.expand ? styles.multi: styles.answer_container}
+                    placeholder={"Start typing..."} onChangeText={value => this.handleChange(value)} 
+                    multiline={this.state.expand} onFocus={()=> this.setState({expand: true})}
+                    onBlur={()=> this.setState({expand: false})} value={this.state.reply}/> }
+                    { this.state.expand ? 
+                        <TouchableOpacity onPress={this.handleSend} disabled={this.state.disabled} 
+                        style={[styles.send_button, {backgroundColor: send_color}]}>
+                        <Icon name='paper-plane' size={20} color='white'/>
+                        </TouchableOpacity>
+                        : <View></View> }
+            </View>
+        );
+    }
+}
+
+export default class ResponseList extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            responses: props.responses,
+            //nav: props.nav
+        };
+    }
+
+    render(){
+        let response_list = [];
+        let res = this.state.responses;
+        for(let i = 0; i < res.length; i++){
+            response_list.push(<Response response={res[i]['response']} 
+            username={res[i]['user']} response_id = {res[i]['id']} key={res[i]['id']}/>);
+        }
+        return(
+            <ScrollView style={{backgroundColor: '#d0faaf'}}>
+                {response_list}
+            </ScrollView>
+        );
+    }
+}
+
+const styles = StyleSheet.create({
```

Comment:
Even if you guys keep styles here, I think you need a more general place for colors. We want every page of the app to share the same colors, after all. 

For that reason, I want to again strongly urge you guys to reconsider moving your styles to a style folder, so that you can maintain style consistency throughout.